### ISSUE TITLE
Compare results of Gas Price Strategy of Raiden

### ISSUE THREAD

**Issue by fredo** [raiden-network/raiden-wizard#195]
@palango also worked on a new gas price strategy to get faster transactions. A good reference here is the value of `ethgasstation` `fast`. 

For the wizard it is also of interest how long it takes to generate the gas price. It seems that it correlates linearely to the sample size

- 120 Blocks =~ 30 sec
- 25 Blocks =~ 7-8 sec

For the wizard I think it is of interest to have a fast calculation for a good user experience as well as fast transaction speed. As a third priority I would say to try lower the gas price to save money for the user. 

Priorities for finding the right gas price strategy: 

1) fast transaction speed
2) fast calculation of gas price
3) save money for the user

This is just my opinion and open for discussion here

[Reference PR in Raiden] (https://github.com/raiden-network/raiden/pull/6135/files)

**Comment by palango** [comment]
> For the wizard it is also of interest how long it takes to generate the gas price. It seems that it correlates linearely to the sample size
> -    120 Blocks =~ 30 sec
>  -  25 Blocks =~ 7-8 sec

This is because the strategies need to download the whole block information with all transactions. We cannot avoid that for the first transaction (only decrease the number of sampled blocks) but cache it, e.g. https://web3py.readthedocs.io/en/stable/middleware.html#web3.middleware.construct_simple_cache_middleware

For the priorities I agree.

**Comment by Dominik1999** [comment]
In the demo of 12th of May with @taleldayekh the first transaction (send ETH to new Account) had a gas price of 1 GWEI. That did not work.

**Comment by fredo** [comment]
Yes, it was a bug and is already fixed. It needs to be tested better though

**Comment by fredo** [comment]
Using a web3 gas price strategy with sample size of 25 still takes 8 seconds which significantly decreases eser experience in the current model. to fix this we would need to implement a global web3 object to be able to cache.

**Comment by palango** [comment]
> sing a web3 gas price strategy with sample size of 25 still takes 8 seconds which significantly decreases eser experience in the current model. to fix this we would need to implement a global web3 object to be able to cache.

What about using the RPC gas price and add a margin of X%?
